.storybook/main.js

```diff
@@ -26,6 +26,11 @@ module.exports = {
             use: ['asciidoc-loader'],
             include: path.resolve(__dirname, '../'),
           },
+          {
+            test: /\.html$/i,
+            loader: 'ignore-loader',
+            include: /node_modules\/codemirror/,
+          },
```

Comment:
👍🏻 Good for now, it will maybe need an update when Storybook will use webpack 5 by default. Wait & see